src/components/Votes/ProposalVotesList/ProposalSingleVote.tsx

```diff
@@ -88,7 +111,32 @@ export function ProposalSingleVote({
                 )}
               </HStack>
               <HStack alignItems="items-center">
-                <TokenAmountDisplay amount={vote.weight} />
+                <TooltipProvider>
```

Comment:
Same feedback about how the hover property of this token amount is not really obvious.

Reply:
FWIW it's not your fault, I think we just need better design on this.